website/public/js/controllers/rootCtrl.js

```diff
@@ -36,6 +36,8 @@ habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$
     $rootScope.toJson = angular.toJson;
     $rootScope.Payments = Payments;
 
+    $http.defaults.headers.common['x-client'] = "habitica-web";
```

Comment:
$http.defaults should be configured only once and then it'll be applied to all requests, it's not necessary to add it twice